ui/apps/platform/src/Containers/Integrations/utils/integrationUtils.ts

```diff
@@ -74,3 +76,70 @@ const getTimes = () => {
 };
 
 export const timesOfDay = getTimes();
+
+// Utilities for image integrations which can have either or both of two categories.
+
+// Categories alternatives correspond to mutually exclusive toggle group items.
+type CategoriesAlternatives<
+    C0 extends ImageIntegrationCategory,
+    C1 extends ImageIntegrationCategory
```

Comment:
I remember when variables could only be two letters long. 👴 

May be better to use more descriptive type names.